moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1981,10 +2055,58 @@ double moveit::core::RobotState::testJointSpaceJump(const JointModelGroup* group
   return percentage;
 }
 
+double moveit::core::RobotState::testJointSpaceJump(const JointModelGroup* group, std::vector<RobotStatePtr>& traj,
+                                                    double revolute_jump_threshold, double prismatic_jump_threshold)
+{
+  double percentage = 1.0;
+  bool still_valid = true;
+  const std::vector<const JointModel*>& joints = group->getActiveJointModels();
+  for (std::size_t i; i < traj.size() - 1 && still_valid; ++i)
```

Comment:
use better var names than i, j

Reply:
Use for (variable : container) syntax provided by C++11.

Reply:
+1 to C++11 here